src/app/modules/time-entries/pages/time-entries.component.spec.ts

```diff
@@ -387,7 +387,7 @@ describe('TimeEntriesComponent', () => {
       uri: 'http://testing.is.fun',
       activity_id: 'sss',
       project_id: 'id',
-      start_date: new Date(),
+      start_date: new Date(new Date().setSeconds(0)),
```

Comment:
I think that this is not so clear when reading the code. Maybe, you can improve like:
```suggestion
      const defaultSeconds = 0
      const currentDate = new Date();
      currentDate.setSeconds(0);
      
      start_date: currentDate,
```